.gitignore

```diff
@@ -9,7 +9,8 @@ gradlew.bat
 .project
 .settings/
 .clang-format
-.vimrc
+.*rc
+.*.env
```

Comment:
should we add any of these to the `.gitignore` generated with `ssc init`?

Reply:
yeah probably `.ssc.env` and `.sscrc` more explicitly though

Reply:
@jwerle 
> should we add any of these to the `.gitignore` generated with `ssc init`?

I just pushed this change.

> yeah probably `.ssc.env` and `.sscrc` more explicitly though

Note that in repo .gitignore, we were originally ignoring `.vimrc` - We might as well blanket ignore in `.gitignore` template as well?

Reply:
We shouldn't blanket ignore all `.*rc` file

Reply:
> We shouldn't blanket ignore all `.*rc` file

Have reverted that.